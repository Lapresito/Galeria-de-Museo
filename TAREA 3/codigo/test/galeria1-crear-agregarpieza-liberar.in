crearGaleria 01/01/2024
crearFecha 01/01/2020 
crearPieza 1 Pieza1 Autor1 Apellido1
agregarPiezaGaleria
crearFecha 02/02/2020 
crearPieza 2 Pieza2 Autor2 Apellido2
agregarPiezaGaleria
crearFecha 03/03/2020 
crearPieza 3 Pieza3 Autor3 Apellido3
agregarPiezaGaleria
crearFecha 04/04/2020 
crearPieza 4 Pieza4 Autor4 Apellido4
agregarPiezaGaleria
liberarGaleria
Fin