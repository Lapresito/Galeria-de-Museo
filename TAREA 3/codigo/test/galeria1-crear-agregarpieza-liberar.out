Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Galeria creada con exito.
2> Fecha creada en forma exitosa.
3> Pieza creada en forma exitosa.
4> La pieza fue agregada a la galeria con exito.
5> Fecha creada en forma exitosa.
6> Pieza creada en forma exitosa.
7> La pieza fue agregada a la galeria con exito.
8> Fecha creada en forma exitosa.
9> Pieza creada en forma exitosa.
10> La pieza fue agregada a la galeria con exito.
11> Fecha creada en forma exitosa.
12> Pieza creada en forma exitosa.
13> La pieza fue agregada a la galeria con exito.
14> Galeria liberada con exito.
15> Fin.
